src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import java.util.Scanner;
+
+import java.io.FileNotFoundException;
+
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+import duke.parser.Parser;
+import duke.storage.Storage;
+
+import java.io.IOException;
+
+import duke.exception.DukeException;
+
+public class Duke {
+
+    private TaskList taskList;
+    private String path;
+
+    public Duke(String path) {
+        this.path = path;
+        this.taskList = new TaskList();
+    }
+
+    public void runDuke() {
+        Ui.greeting();
+        try {
+            Storage.loadData(this.taskList, this.path);
+            System.out.println("\tYour previous tasks have been loaded!");
+        } catch (FileNotFoundException e) {
+            System.out.println("\tWelcome new user!");
+        }
+
+        boolean quit = false;
```

Comment:
Boolean naming should follow the Java coding standard.
```suggestion
    boolean needsToQuit = false;
```